File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldChipSetDateTime.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class StudioFieldChipSetDateTime extends StudioFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public Date[] defaultValue;

  @Nullable
  public String displayDateFormat;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldChipSetDateTime.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import java.util.Date
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.dto.StudioFieldEditable

interface StudioFieldChipSetDateTime : StudioFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: Array<String>?
  val displayDateFormat: String?
}